.travis.sh

```diff
@@ -0,0 +1,10 @@
+#! /bin/bash
+
+set -e -x
+
+make -C control check
```

Comment:
Why cannot this be part of the generic yast-travis-ruby? Wouldn't it be better to file this under `rake check:syntax`?